tests/unittests.py

```diff
@@ -752,6 +752,71 @@ def test_meta_dataset(self):
         dataset1c = fo.load_dataset(dataset_name)
         self.assertIs(dataset1c, dataset1)
 
+    @drop_datasets
+    def test_merge_samples(self):
+        dataset1 = fo.Dataset()
+        dataset2 = fo.Dataset()
+
+        common_filepath = "/path/to/image.png"
+        filepath1 = "/path/to/image1.png"
+        filepath2 = "/path/to/image2.png"
```

Comment:
This introduced a test failure on Windows - paths are normalized internally, so they will not be equal to these on Windows.
I'll try to fix this in #547, but evidently nobody checked before this PR was merged. Maybe we should make the tests a required check? https://docs.github.com/en/github/administering-a-repository/enabling-required-status-checks